karapace/auth.py

```diff
@@ -87,24 +88,31 @@ async def start_refresh_task(self, stats: StatsClient) -> None:
 
         async def _refresh_authfile() -> None:
             """Reload authfile, but keep old auth data if loading fails"""
-            try:
-                # Poll delay for fallback mechanism if system notifications cannot be used.
-                async for changes in awatch(
-                    self._auth_filename, poll_delay_ms=5000, stop_event=self._refresh_auth_awatch_stop_event
```

Comment:
Do we know for sure that `poll_delay_ms` has no effect without `force_polling`? That seemed ambiguous in docs.

Reply:
From the Rust code, it seems like it's not used if `force_polling` is not True

Reply:
✅👍